metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -707,6 +850,182 @@ def filter_nodes(self, all_nodes: List[DBTNode]) -> List[DBTNode]:
 
         return nodes
 
+    @staticmethod
+    def _to_schema_info(schema_fields: List[SchemaField]) -> SchemaInfo:
+        return {column.fieldPath: column.nativeDataType for column in schema_fields}
+
+    def _infer_schemas_and_update_cll(self, all_nodes_map: Dict[str, DBTNode]) -> None:
```

Comment:
A little overview on what this function is doing and the steps it takes would be nice

Reply:
added some details